great-idea/css/index.css

```diff
@@ -63,4 +63,496 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+
+
+
+/* Your code starts here! */
+
+
+
+/* Makes Everything Centered */
+
+.container {
+	width: 80%;
+	margin: 0 auto;
+	overflow: hidden;
+	
+ }
+ 
+ 
+ 
+ 
+
+
+ /* SERVICES PAGE */
+
+
+ /* Header, Nav, and Logo */
+ 
+
+ header {
+	 display: flex;
+	 /* border: 1px solid black; */
+	 justify-content: space-between;
+	 align-items: center;
+	 padding: 20px;
+ }
+
+ header nav {
+	 /* border: 1px solid red;	 */
+	 display:flex;
+	 
+ }
+
+ header nav a {
+	 margin-right: 40px;
+	 text-decoration: none;
+	 color: black;
+	 padding: 10px 20px;
+	 border: 1px solid gray;
+	 border-radius: 5px;
+	 transition: 0.2s ease-in;	
+	 /* border: 1px solid blue; */
+ }
+
+
+ header nav a:hover {
+	background-color: aquamarine;
+	border: 1px solid black;
+	font-weight: bold;
+	
+ }
+
+
+
+ /* SERVICES IMAGE */
+
+ .services-header-img {
+	 width: 100%;
+	
+ }
+
+p {
+	line-height: 1.5;
+}
+
+
+ /* Services section */
+
+.services {
+ margin: 40px 0;
+ padding-bottom: 40px;
+ border-bottom: 2px solid black;
+}
+
+
+
+/* Wrapper section */
+
+
+.wrapper {
+	
+	display:flex;
+	flex-wrap: wrap;
+	justify-content: center;
+	align-items: center;
+	
+	
+	
+}
+
```

Comment:
Wrapping all of the six cards in one container gives us some weird behaviour. Try wrapping each horizontal row of two cards in a flex container, and not using flex-wrap. Try resizing your window a few times and see if your layout breaks.